lupa/tests/test.py

```diff
@@ -591,6 +594,58 @@ def test_table_from_table_iter_indirect(self):
         self.assertEqual(list(table2.keys()), [1, 2, 3])
         self.assertEqual(set(table2.values()), set([1, 2, "foo"]))
 
+    def test_table_from_nested_dict(self):
+        data = {"a": {"a": "foo"}, "b": {"b": "bar"}}
+        table = self.lua.table_from(data, recursive=True)
+        self.assertEqual(table["a"]["a"], "foo")
+        self.assertEqual(table["b"]["b"], "bar")
+        self.lua.globals()["data"] = table
+        self.lua.eval("assert(data.a.a=='foo', 'failed')")
```

Comment:
Is there a test where we make sure that this raises an exception if the condition fails, so that the test actually fails?

Reply:
Ok, I would add it later